docs/modules/configuration/pages/dynamic-config-overview.adoc

```diff
@@ -1,12 +1,50 @@
-= Updating Member Configuration at Runtime
+= Overview
```

Comment:
3 reasons:

1. I want to see same links text on the left navigation page and title.
2. Overview better summatizes what is on this page IMO.
3. Update is the keyword for configuration update and it could be a little misleading.

Reply:
Perhaps we just title this page 'Dynamic Configuration'?

We can control what is displayed in the left navigation separately. See https://github.com/hazelcast/hz-docs/blob/main/docs/modules/configuration/partials/nav.adoc

Reply:
Yes I know, but I just wasnted both of them to be the same and overivew made more sense at that time. Dynamic Configuration is also good but then we will have double Dynamic Configuration on the nav.